mods/lm-close-windbg-file-changed.wh.cpp

```diff
@@ -0,0 +1,77 @@
+// ==WindhawkMod==
+// @id           lm-close-windbg-file-changed
+// @name         Close WinDbg's file changed dialog
+// @description  Automatically rejects (or accepts) the dialog from WinDbg telling that a file is changed
+// @version      1.0
+// @author       Mark Jansen
+// @github       https://github.com/learn-more
+// @twitter      https://twitter.com/learn_more
+// @include      windbg.exe
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+# Close WinDbg's file changed dialog
+This mod automatically dismisses the popup from WinDbg notifying about a file change.
+When automating WinDbg, this dialog blocks progress.
+*/
+// ==/WindhawkModReadme==
+
+// ==WindhawkModSettings==
+/*
+- Accept: 0

```

Comment:
FYI, you can use `false` here to have a checkbox instead of a number field. As far as the mod is concerned, the value will be an integer of either `0` or `1`, so the code needs no changes.

Reply:
Sure, I'll change this and add a screenshot later today or tomorrow.